gems/aws-sdk-core/lib/aws-sdk-core/static_credentials_provider.rb

```diff
@@ -10,6 +10,17 @@ module AWS::SDK::Core
   #     )
   #     ec2 = AWS::SDK::EC2::Client.new(credentials_provider: provider)
   class StaticCredentialsProvider < Hearth::IdentityProvider
+    # Initializes an instance of StaticCredentialsProvider using code options.
+    def self.from_code(config)
```

Comment:
nit: maybe call from_config instead of from code?

Edit - nvm, Is the new cross sdk standard calling values taken from config "code" now?

Reply:
This is from code (config) not shared config